src/web/upload/upload_bot_new.py

```
@@ -0,0 +1,66 @@
+
+import requests
+from pathlib import Path
+import logging
+
+
+from web.auth.wiki_site import Site, FileExists, InsufficientPermission
+
+logger = logging.getLogger(__name__)
+
+
+def upload_file(file_name, file_path, site: Site=None, summary=None):
```

Flake8 (7.3.0)

[error] missing whitespace around parameter equals

(E252)